frontend/src/app/modules/user-profile/user-details/user-details/user-details.component.ts

```diff
@@ -89,6 +90,10 @@ export class UserDetailsComponent extends BaseComponent implements OnInit {
     onSubmit() {
         const userDetails = <IUserInfo> this.detailsForm.value;
 
+        const birthDateMoment = moment(userDetails.birthDate, 'YYYY-MM-DD').add(1, 'day');
```

Comment:
Can you maybe find the reason? As it maybe more global problem

Reply:
But how it happens, that on register page it works and on user details - doesn't?